### PR TITLE
CLI uninstall instructions

### PR DESCRIPTION
**Pull request by rosemarymarano**
CLI uninstall instructions

### CODE REVIEW

File: content/en/docs/setup/uninstall/uninstall.md [hunk 1]
```diff
@@ -2,19 +2,46 @@
 title: "Uninstall"
 linkTitle: "Uninstall"
 description: "How to uninstall Verrazzano"
-weight: 6
+weight: 5
 draft: false
 ---
 
+
 To delete a Verrazzano installation, delete the Verrazzano custom resource you used to
```
Comment (bobdonat) [inline content/en/docs/setup/uninstall/uninstall.md:10]: I don't think this line is needed.  For the CLI flavor the user does not delete the verrazzano custom resource.  For the kubectl flavor they do delete the verrazzano custom resource so maybe you only need to mention this in the kubectl section.

File: content/en/docs/setup/uninstall/uninstall.md [hunk 2]
```diff
@@ -2,19 +2,46 @@
...
+   ...
+   ```
+{{< /tab >}}
+{{< tab tabName="kubectl" >}}
+<br>
 
-The following example starts a deletion of a Verrazzano installation in the background, and then
-uses the `kubectl logs -f` command to tail the Console output of the pod performing the uninstall:
+The following example starts a deletion of a Verrazzano installation in the background and then
+uses the `kubectl logs -f` command to tail the output of the pod performing the uninstall.
```
Comment (bobdonat) [inline content/en/docs/setup/uninstall/uninstall.md:44]: Should the steps in the kubectl section be numbered to be consist with the CLI section

File: content/en/docs/setup/uninstall/uninstall.md (hunk 1 above)
Comment (mgianatagh) [reply content/en/docs/setup/uninstall/uninstall.md:10]: correct, this belongs in the kubectl tab

Comment (rosemarymarano) [reply content/en/docs/setup/uninstall/uninstall.md:10]: Thanks. @stevshap will make the technical edits.

File: content/en/docs/setup/uninstall/uninstall.md (hunk 2 above)
Comment (rosemarymarano) [reply content/en/docs/setup/uninstall/uninstall.md:44]: @bobdonat Oh, I see. The steps are written as code comments. Yes, I will elevate them to numbered steps. Thank you.

File: content/en/docs/setup/uninstall/uninstall.md [hunk 3]
```diff
@@ -2,29 +2,61 @@
...
+1. Get the name of the Verrazzano custom resource.
+
+   ```shell
+   $ MYVZ=$(kubectl  get vz -o jsonpath="{.items[0].metadata.name}")
+   ```
+2. Delete the Verrazzano custom resource.
+
+   ```shell
+   $ kubectl delete verrazzano $MYVZ --wait=false
+   $ kubectl logs -n verrazzano-install \
```
Comment (bobdonat) [inline content/en/docs/setup/uninstall/uninstall.md:55]: I think this should be step 3.  Something like "View the uninstall log and wait for uninstall to complete".

Comment (rosemarymarano) [reply content/en/docs/setup/uninstall/uninstall.md:55]: Yep, right again. I'll fix it.

**Review by stevshap — APPROVED** [review]
Approved

**Pull request merged**